kratos/solving_strategies/convergencecriterias/convergence_criteria.h

```diff
@@ -14,20 +14,15 @@
 #if !defined(KRATOS_NEW_CONVERGENCE_CRITERIA )
 #define  KRATOS_NEW_CONVERGENCE_CRITERIA
```

Comment:
ouch, why "NEW" ?

Reply:
It was that way when I come here

Reply:
can u change that as a part of the cleanup?